internal/connector/internal/api/admin/private/api/openapi.yaml

```diff
@@ -133,6 +133,48 @@ paths:
       summary: Returns a list of connector clusters
       tags:
       - Connector Clusters Admin
+  /api/connector_mgmt/v1/admin/kafka_connector_clusters/{connector_cluster_id}:
```

Comment:
Hmm I think this is coming from running `make generate code/fix`

I suggest that we open a separate PR for that.

/cc @lburgazzoli @dhirajsb @valdar do you know why we are getting this? Is it something that has already been fixed and a rebase will be enough?

Reply:
@VanillaSpoon can you rebase the PR once https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1030 is merged?